aws-datastore/src/main/java/com/amplifyframework/datastore/network/SyncEngine.java

```diff
@@ -151,7 +159,12 @@ private void startDrainingChangeJournal() {
                 .flatMapSingle(this::publishToNetwork)
                 .flatMapSingle(storageItemChangeJournal::remove)
                 .subscribe(
-                    processedChange -> LOG.info("Change processed successfully! " + processedChange),
+                    processedChange -> {
+                        LOG.info("Change processed successfully! " + processedChange);
```

Comment:
Is this just for local debugging and should we remove?

Reply:
Hm, I should change it from `.info` to `.debug`. Since the default is to only print info's or better, debug would effectively turn if off (unless somebody changes the log level.)